mslib/mscolab/models.py

```diff
@@ -39,17 +39,28 @@ class User(db.Model):
     id = db.Column(db.Integer, primary_key=True, autoincrement=True)
     username = db.Column(db.String(255))
     emailid = db.Column(db.String(255), unique=True)
-    password = db.Column(db.String(255), unique=True)
```

Comment:
users should not but it can be they have chooen a password identical to some other user. Telling this is not a good idea.